EIPS/eip-draft-extbalance.md

```diff
@@ -0,0 +1,37 @@
+---
+eip: 2604
+title: EXTBALANCE rename and conditional pricing
+author: wjmelements.eth <wjmelements@gmail.com> (@wjmelements)
```

Comment:
```suggestion
author: wjmelements.eth (@wjmelements)
```

Has to be one or the other, and we strongly prefer at least one author has a github username so EIPs can auto-merge correctly.